cpmpy/solvers/pysat.py

```diff
@@ -81,7 +81,7 @@ def solvernames():
         from pysat.solvers import SolverNames
         names = []
         for name, attr in vars(SolverNames).items():
-            if not name.startswith('__') and isinstance(attr, tuple):
+            if not name.startswith('__') and isinstance(attr, tuple) and not 'crypto' in name:
```

Comment:
can you add why you are blocking it? maybe also add its name in full?
are we sure this is not an upstream bug btw, maybe we should report it while also (temporarily?) blocking it here?

Reply:
linked the github issue